docs/source/plugins/06_solver_hooks.rst

```diff
@@ -3,10 +3,10 @@
 Solver Hooks
 ============
 
-The present section describes all solver `Hooks` available on |alfasim| plugin infrastructure.
-Solver `Hooks` are |alfasim|'s pre-defined functions that allows the plugin developer to modify/extend |alfasim|'s Solver.
-As already informed in :ref:`Quick Start <quick-start-section>` section once created a plugin using ``new`` option
-on |sdk|'s CLI a new file named ``plugin.c`` will be available to implement those `hooks`.
+The present section describes all solver `Hooks` available in |alfasim| plugin infrastructure.
+Solver `Hooks` are |alfasim|'s pre-defined functions that allow plugin developer to modify or extend |alfasim|'s Solver.
+As already informed in :ref:`Quick Start <quick-start-section>` section, once created a plugin using ``new`` option
+in |sdk|'s CLI a new file named ``plugin.c`` will be available to implement those `hooks`.
```

Comment:
This seems to be not up to date. Currently, the file created is `plugin.cpp` and not `plugin.c`. Or am I missing something?

Reply:
You're right. Fixed.